### ISSUE TITLE
UPC_process.py: Need to set detached label field (when needed)

### ISSUE THREAD

**Issue by USGSsakins** [USGS-Astrogeology/PDS-Pipelines#297]
When creating or updating `datafiles` entries, the detached label field should be populated with any detached label (ie. ending in .lbl).  The getkeywords to determine if there is a detached label will show the image file the label references (or it will be the same filename as the label).  Both of these fields should be URLs.

https://github.com/USGS-Astrogeology/PDS-Pipelines/blob/d88df281472f9065f193d7fb0dc63646ab57971e/pds_pipelines/UPC_process.py#L277

**Comment by dpmayerUSGS** [comment]
For PDS3 products, UPC can determine if the label it is reading is detached or not by looking at the `^IMAGE` keyword. For products with attached labels, the value of this keyword should be a single value of positive integer type, indicating the record number that the data area starts at:
```
^IMAGE = 2
```
For products with *detached* labels, the keyword will have a value that is a 2 item list containing the name of the file containing the data and the record number in that file that the data starts at:
```
^IMAGE = ("9500R.IMG",7)                                                      
```

**Comment by AustinSanders** [comment]
Closed in #310
